manager/src/main/java/com/redhat/service/smartevents/manager/ProcessorServiceImpl.java

```diff
@@ -295,7 +297,10 @@ public Long getProcessorsCount(String bridgeId, String customerId) {
     @Transactional
     @Override
     public ListResult<Processor> getUserVisibleProcessors(String bridgeId, String customerId, QueryProcessorResourceInfo queryInfo) {
-        Bridge bridge = bridgesService.getReadyBridge(bridgeId, customerId);
+        Bridge bridge = bridgesService.getBridge(bridgeId, customerId);
+        if (!isManagedResourceActionable(bridge)) {
```

Comment:
@manstis I'm just wondering if we need this check at all? Is there any harm in allowing a user to list the Processors on a bridge in `ACCEPTED`, `PREPARING` or `PROVISIONING` status? 

I completely agree the check is useful when the user tries to provision a `Processor`, but I'm not sure it really matters when they're performing a list operation. As/when #875 gets merged, we'd need to support that anyway as a user should be able to list the `Processors` on their `Bridge` when it's being updated.

Reply:
@robpblake The previous implementations call to `bridgeService.getReadyBridge(..)` threw the `BridgeLifecycleException` if the `Bridge` was not in `READY` state. Keeping this check ensures the User's experience of this API method is unchanged. i.e. it will still throw the exception if the `Bridge` is not `READY` (or `FAILED`). See previous discussion about this [above](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/902#discussion_r911785976).

Reply:
@manstis OK thanks. We still have discussions to resolve around #875 so I'm getting ahead of myself and making assumptions on future Bridge lifecycle. 

cheers,
Rob